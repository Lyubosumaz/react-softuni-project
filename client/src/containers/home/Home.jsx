import './home.css';

export default function Home() {
    return (
        <div className="main-container background-image" name="home">
            <h1>Home</h1>
        </div>
    );
}
