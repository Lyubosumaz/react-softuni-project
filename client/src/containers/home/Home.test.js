import React from 'react';
import Home from './Home';
import { shallow } from 'enzyme';

it('Renter Home component', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toBeTruthy();
});

it('Checks for html changes', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
});
