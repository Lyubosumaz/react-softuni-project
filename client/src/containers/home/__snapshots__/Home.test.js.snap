// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should checks for html changes 1`] = `
<div
  className="main-container background-image"
  name="home"
>
  <h1>
    Home
  </h1>
</div>
`;
